lib/blocs/companies_cubit/companies_cubit.dart

```diff
@@ -0,0 +1,66 @@
+import 'dart:async';
+
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:twake/blocs/companies_bloc/companies_bloc.dart';
+import 'package:twake/blocs/notification_bloc/notification_bloc.dart';
+import 'package:twake/blocs/profile_bloc/profile_bloc.dart';
+import 'package:twake/models/globals/globals.dart';
+import 'package:twake/repositories/collection_repository.dart';
```

Comment:
No need to reuse collections_repository, just create a specialized repository for this cubit, analogous to WorkspacesRepository